src/coreclr/src/inc/dacvars.h

```diff
@@ -222,7 +221,6 @@ DEFINE_DACVAR(ULONG, SIZE_T, dac__gFCallMethods, ::gFCallMethods)
 DEFINE_DACVAR(ULONG, PTR_SyncTableEntry, dac__g_pSyncTable, ::g_pSyncTable)
 #ifdef FEATURE_COMINTEROP
 DEFINE_DACVAR(ULONG, UNKNOWN_POINTER_TYPE, dac__g_pRCWCleanupList, ::g_pRCWCleanupList)
-DEFINE_DACVAR(ULONG, BOOL, RCWWalker__s_bIsGlobalPeggingOn, RCWWalker::s_bIsGlobalPeggingOn)
```

Comment:
>DEFINE_DACVAR [](start = 0, length = 13)

This is actually still desired since we handle the global pegging logic in interoplib.

Reply:
To my understanding, the pegging logic only applies to WinRT. In which case, the built in RCW/CCW system doesn't need to understand pegging any more. As a result, the DAC doesn't need to know about pegging since it only uses this variable to handle telling the debugger if a given built-in RCW/CCW is pegged.

If my assumption is incorrect, I'll add pegging back to the built-in RCW/CCW system and re-expose this variable to the DAC from interoplib.

Reply:
No worries. I wasn't sure why the DAC needed it so I kept it. If that is why, then you are correct it can be removed. Thanks!